Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -37,44 +44,53 @@ def get_geo_df(data: pd.DataFrame,
     :param data: DataFrame of values and geographies
     :param geo_value_col: name of column containing values of interest
     :param geo_type_col: name of column containing geography type
+    :param join_type: Type of join to do between input data (left side) and geo data (right side). \
```

Comment:
I think I used the \ because it there was some warning when building without it and the first answer from google had the `\`. Two spaces seems much cleaner, so will change to that.